runtime/jcl/common/shared.c

```diff
@@ -1890,8 +1890,54 @@ Java_com_ibm_oti_shared_SharedClassStatistics_freeSpaceBytesImpl(JNIEnv* env, jo
 #endif
 }
 
+jstring JNICALL
+Java_com_ibm_oti_shared_SharedClassStatistics_cacheDirImpl(JNIEnv* env, jobject thisObj)
+{
+#if defined(J9VM_OPT_SHARED_CLASSES)
+	J9JavaVM* vm = ((J9VMThread*)env)->javaVM;
```

Comment:
nit: there should be a space before `*` (but not after).

Reply:
There should be one before each `*` here, but I won't push.

Reply:
Updated it.